.gitignore

```
@@ -109,3 +109,5 @@ ENV/
 
 # experiments
 private_*
+
+df.parquet
```

Use the same as in #160 to avoid merge conflicts.
```suggestion
*.parquet
```